solidity/contracts/test/deposit/TestDeposit.sol

```diff
@@ -123,10 +123,6 @@ contract TestDeposit is Deposit {
         self.keepAddress = _keepAddress;
     }
 
-    function getKeepAddress() public view returns (address) {
-        return self.keepAddress;
-    }
-
```

Comment:
You have getters for undercollateralized percentages below as well.

Reply:
took care of them